src/algo/thermal-loop/l500-thermal-loop.h

```diff
@@ -0,0 +1,80 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <vector>
+#include "../../types.h"
+
+namespace librealsense {
+namespace algo {
+namespace thermal_loop {
+namespace l500 {
+
+using fx_fy = std::pair< double, double >;
+
+#pragma pack( push, 1 )
+
+class l500_thermal_loop
+{
+public:
+    struct rgb_thermal_calib_info
+    {
+        static const int table_id = 0x317;
+        static const int resolution = 29;
```

Comment:
Needs documentation

Reply:
'temp_step'  renamed to  'temp_interval'
and its 29 bins between min and max temps so its divides to 30 equals intervals.